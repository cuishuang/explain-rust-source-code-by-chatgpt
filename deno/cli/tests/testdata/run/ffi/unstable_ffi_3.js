const { op_ffi_call_ptr_nonblocking } = Deno[Deno.internal].core
  .ensureFastOps();
op_ffi_call_ptr_nonblocking(null, {
  name: null,
  parameters: [],
  result: "void",
}, []);
