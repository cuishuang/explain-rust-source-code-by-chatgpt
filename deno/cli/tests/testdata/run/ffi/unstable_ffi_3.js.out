Unstable API 'Deno.UnsafeFnPointer#call'. The `--unstable-ffi` flag must be provided.
